### PR TITLE
feat(archlinux): add cleanup aliases to `yaourt` and `yay`

### PR DESCRIPTION
**Pull request by mralaminahamed**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

**Comment by mralaminahamed** [comment]
Hi friend, I miss two option of yay and yourt  for clean cache. I want to add archlinux package for everyone to use it.

File: plugins/archlinux/archlinux.plugin.zsh [hunk 1]
```diff
@@ -171,6 +171,8 @@ if (( $+commands[yaourt] )); then
   alias yamir='yaourt -Syy'
   alias yaupd="yaourt -Sy"
   alias upgrade='yaourt -Syu'
+  alias yacun='yay -Yc'
+  alias yacf='yay -Scc'
```
Comment (mcornella) [inline plugins/archlinux/archlinux.plugin.zsh:175]: These two use `yay` even though they're in the `yaourt` block.

File: plugins/archlinux/archlinux.plugin.zsh [hunk 2]
```diff
@@ -191,5 +193,7 @@ if (( $+commands[yay] )); then
   alias yamir='yay -Syy'
   alias yaupd="yay -Sy"
   alias upgrade='yay -Syu'
+  alias yacun='yay -Yc'
+  alias yacf='yay -Scc'
```
Comment (mcornella) [inline plugins/archlinux/archlinux.plugin.zsh:197]: You should document these in the plugin README.

**Review by mcornella — CHANGES_REQUESTED** [review]

**Comment by mcornella** [comment]
I assume you've made these changes via the GitHub website. To make more changes, you can [go to the `patch-1` branch](https://github.com/mralaminahamed/ohmyzsh/tree/patch-1) and edit the files there, then make a commit in the same branch.

Also use a more descriptive Pull Request title, it's not obvious what these aliases do. Thanks!

File: plugins/archlinux/archlinux.plugin.zsh (hunk 2 above)
Comment (mralaminahamed) [reply plugins/archlinux/archlinux.plugin.zsh:197]: ok, i do this.

**Review by mcornella — APPROVED** [review]

**Pull request merged**